packages/react-components/react-color-picker-preview/library/src/components/AlphaSlider/useAlphaSliderState.ts

```diff
@@ -26,16 +28,21 @@ export const useAlphaSliderState_unstable = (state: AlphaSliderState, props: Alp
 
   const inputOnChange = state.input.onChange;
 
-  const _onChange: React.ChangeEventHandler<HTMLInputElement> = useEventCallback(ev => {
-    const newValue = Number(ev.target.value);
-    setCurrentValue(clamp(newValue, min, max));
-
-    if (inputOnChange && inputOnChange !== (onChange as unknown as React.ChangeEventHandler<HTMLInputElement>)) {
-      inputOnChange(ev);
-    } else if (onChange) {
-      onChange(ev, { type: 'change', event: ev, value: newValue });
-    }
-  });
+  const _onChange: React.ChangeEventHandler<HTMLInputElement> = useEventCallback(
+    mergeCallbacks(
```

Comment:
`mergeCallbacks()` seems to be redundant there, you can do:

```
      event => {
        const newValue = Number(event.target.value);
        setCurrentValue(clamp(newValue, min, max));
        inputOnChange && inputOnChange(event);
        onChange && onChange(event, { type: 'change', event, value: newValue, channel });
        ctxOnChange(event, {
          value: Number(event.target.value),
          channel,
        }),
```

Reply:
fixed